copilot/fastapi/app/main.py

```
@@ -0,0 +1,153 @@
+    except Exception as e:
+        health_status["services"]["argo_cd"] = {
+            "status": "unhealthy",
+            "error": str(e)
+        }
+        health_status["status"] = "degraded"
+    
+    return health_status
+
+if __name__ == "__main__":
+    import uvicorn
+    uvicorn.run("main:app", host="0.0.0.0", port=8000, reload=True)
```

Ruff (0.8.2)

Possible binding to all interfaces

(S104)